internal/ui/table.go

```diff
@@ -202,9 +203,11 @@ func (t *Table) doUpdate(data *render.TableData) {
 		cols = t.viewSetting.Columns
 	}
 	custData := data.Customize(cols, t.wide)
-	if t.viewSetting != nil && t.viewSetting.SortColumn != "" {
+	// The sortColumn settings in the configuration file can only be used once
+	if t.viewSetting != nil && t.viewSetting.SortColumn != "" && !t.configSort {
```

Comment:
I think we will have a problem here ie if the user changes the view config to adjust the sort col, she will never see that change while in the view. Imho we want to reset the flag if the view config changes and override the sort col.
Same idea as you had here Jayson but was thinking using something like `manualSort` to indicate the user has physically changed the sort order and reset the flag when the config changes in `ViewSettingsChanged`.
Would this make sense?

Reply:
Indeed, it would be easier to understand using `manualSort` to indicate whether the user has manually modified the sort col. On the other hand, if we reset the flag and override the sort col in `ViewSettingsChanged`, would this not be more strange for the user (who has already manually modified the sort col, but reset it after modifying the configuration)?

Reply:
> I think we will have a problem here ie if the user changes the view config to adjust the sort col, she will never see that change while in the view. Imho we want to reset the flag if the view config changes and override the sort col. Same idea as you had here Jayson but was thinking using something like `manualSort` to indicate the user has physically changed the sort order and reset the flag when the config changes in `ViewSettingsChanged`. Would this make sense?

Indeed, it would be easier to understand using `manualSort` to indicate whether the user has manually modified the sort col. On the other hand, if we reset the flag and override the sort col in `ViewSettingsChanged`, would this not be more strange for the user (who has already manually modified the sort col, but reset it after modifying the configuration)?

Reply:
Thank you for the feedback Jayson! 
With the view configuration in play, when the user tweaks a given resource while k9s is up and in that resource view, k9s will update when the view config file is saved. This applies to columns and sort configs. Thus if manual sort is at play and then the user alters the view config, it seems that if should override the current sort settings so we are consistent. 
Would this make sense?

Reply:
Sorry for the late reply, the flag is now reset after the profile change. Take a look at this PR when you have time. 😊